README.md

```diff
@@ -228,26 +224,37 @@ GTMSessionFetcher *fetcher = [fetcherService fetcherWithURL:userinfoEndpoint];
 
 ### Saving to the Keychain
 
-You can easily save `GTMAppAuthFetcherAuthorization` instances to the Keychain using
-the included `GTMAppAuthFetcherAuthorization+Keychain` category.
+You can easily save `GTMAuthSession` instances to the Keychain using the `GTMKeychainStore` class.
 
 ```objc
+// Create a GIDKeychainStore instance
+GIDKeychainStore *keychainStore =
+    [[GIDKeychainStore alloc] initWithItemName:kGTMAppAuthExampleAuthorizerKey];
+    
+NSError *error;
+
 // Save to Keychain
-[GTMAppAuthFetcherAuthorization saveAuthorization:_authorization
-                                toKeychainForName:kGTMAppAuthExampleAuthorizerKey];
+[keychainStore saveWithAuthSession:self.authSession error:&error];
```

Comment:
@mdmathias we might want to rename the method`saveWithAuth...` -> `saveAuth...`.  Thoughts?

Reply:
@petea agreed. [Changed here](https://github.com/google/GTMAppAuth/pull/207/commits/68f33f07cf37f4904b6e728f2c029b30131df85b).